apps/web/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluentui/react-native": "^0.15.55",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "react": "^16.13.1",
+    "react-art": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-native": "0.62.2",
+    "react-native-web": "^0.12.3",
+    "react-scripts": "3.4.1",
+    "typescript": "~3.7.2",
+    "webpack": "4.42.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
react-scripts are a good way to get started, longer term we probably want to eject as typically the automatic configuration will eventually start working against you. No rush on this, just pointing out that just about everyone ends up ejecting at some point so don't be afraid to consider that as an option in the future if you get stuck.